common/source/docs/common-glossary.rst

```diff
@@ -229,15 +229,15 @@ camera pointed towards.
 The square-wave pulse train used to transmit multiple channels of
 information between some RC transmitters and receivers. Some RC
 receivers provide a PPM output (sometimes referred to as the PPM Sum)
-which can be used on the APM and PX4. Other receivers convert the PPM
+which can be used on Pixhawk-series boards. Other receivers convert the PPM
 signal and only provide PWM signals for each channel.
 
 **PWM**: Pulse Width Modulation. The square-wave signals used in RC
 control to drive servos and speed controllers. There is one PWM signal
 for each channel. The width varies from 1000 to 2000 microseconds,
 depending on the RC manufacturer.
 
-**PX4 (PX4FMU and PX4IO)**: Flight Controller system providing
+**Pixhawk (PX4FMU and PX4IO)**: Flight Controller system providing
```

Comment:
This is actually incorrect because it's referring to the software.  I think we should change this to be just "PX4" and say something like, "a separate flight code stack with a name that is often confused with the Pixhawk hardware".

Reply:
Not my intention. I mean to identify the hardware. I'll do something like "Early hardware revision of the Pixhawk flight controller board"